openedx/core/lib/block_structure/tests/helpers.py

```diff
@@ -147,6 +147,21 @@ def transform(self, usage_info, block_structure):
         pass
 
 
+class MockFilteringTransformer(FilteringTransformerMixin, BlockStructureTransformer):
+    """
+    A mock FilteringTransformerMixin class.
+    """
+    VERSION = 1
+
+    @classmethod
+    def name(cls):
+        # Use the class' name for Mock transformers.
+        return cls.__name__
+
+    def transform_block_filters(self, usage_info, block_structure):
+        return [block_structure.create_universal_filter()]
```

Comment:
Nit: Not technically a mock.  Maybe `StubFilteringTransformer`, or `NoOpFilteringTransformer`?  Cf. http://stackoverflow.com/questions/2665812/what-is-mocking